centml/compiler/server.py

```diff
@@ -82,7 +79,7 @@ async def compile_model_handler(model_id: str, model: UploadFile, inputs: Upload
         return Response(status_code=200)
 
     # This effectively sets the model's status to COMPILING
-    os.makedirs(os.path.join(storage_path, model_id))
+    os.makedirs(os.path.join(config_instance.SERVER_BASE_PATH, model_id))
```

Comment:
why do we need this?

Reply:
If you're asking why we need to call `os.makedirs(...)` it's because this is what sets the compilation status to `COMPILING`. Before this, I only call `os.makedirs()` for the directory `SERVER_BASE_PATH` (i.e. `~/.cache/centml/server`. Here I am creating the directory `{SERVER_BASE_PATH}/{model_id}`